rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/provider/DiscoverFilesAndTypesRuleProvider.java

```diff
@@ -32,9 +36,22 @@ public Configuration getConfiguration(RuleLoaderContext ruleLoaderContext)
     {
         return ConfigurationBuilder.begin()
 
+        // Mark all input paths as ApplicationProjectModel.
```

Comment:
This comment is actually incorrect.

Reply:
Right, this was changed by IDEA refactoring... fixed